src/api.jl

```diff
@@ -41,3 +41,14 @@ function surrogate(x, method::Surrogate)
     sg = surrogenerator(x, method)
     sg()
 end
+
+"""
+    surrogate(x,t, method::Surrogate) → s

```

Comment:
I think the order of the arguments should be `t`, then `s`. I can't think of many contexts where my first argument (x-axis) is signal and my second argument (y-axis) is time, most often the opposite! That is also the order of arguments in the LombScargle package.

Reply:
This is irrelevant now, because I put the t into the method type